pipelines/team-tasks.yml

```diff
@@ -0,0 +1,108 @@
+members: &members
+# Aidan
+- "<@W013S2H8ABU>"
+# Scott
+- "<@W013B9SAXJT>"
+# Daniel
+- "<@W013QLP3YJZ>"
+# Esteban
+- "<@U01LJULP0LS>"
+# Alex
+- "<@W013S7H1JCT>"
+# Rui
+- "<@W012PK4PCR1>"
+# Taylor
+- "<@W011PN57Q8Y>"
+# Muntasir
+- "<@W011EC91F5L>"
+# Navdeep
+- "<@W013K986P7G>"
+# Matthew
+- "<@W013BA1CNMD>"
+# Clara
+- "<@W013B99CVU7>"
+
+resource_types:
+- name: slack-notification
+  type: registry-image
+  source:
+    repository: cfcommunity/slack-notification-resource
+
+- name: static-list-resource
+  type: registry-image
+  source:
+    repository: taylorsilva/static-list-resource
+
+- name: cron-resource
+  type: registry-image
+  source:
+    repository: cftoolsmiths/cron-resource
+
+resources:
+- name: slack-message
+  type: slack-notification
+  source:
+    url: ((slack_hook))
```

Comment:
nit: could we use `((slack_hooks.concourse-private))` instead? it's totally equivalent, but I've been trying to use `slack_hooks` where possible since it's explicit about what channel we send to

Reply:
Will do! I blindly used what I saw in `concourse.yml`, so thanks for sharing!